### ISSUE TITLE
Low accuracy of UCF101 on ResNet18

### ISSUE THREAD

**Issue by bandontseng** [kenshohara/3D-ResNets-PyTorch#76]
Hi,

I followed the command provided in README to train UCF101 on ResNet18 with pretrained model at 200 epochs. However, the accuracy is pretty low. The following blocks are my result of eval_ucf101.py and opts.json. Could you help me to know which part is wrong? Thanks!

```
[INIT] Loaded annotations from validation subset.
        Number of ground truth instances: 3783
        Number of predictions: 37540
[RESULTS] Performance on ActivityNet untrimmed video classification task.
        Error@1: 0.7687020882897171
```

```
{
    "root_path": "./data/ucf101/",
    "video_path": "./data/ucf101/video_jpegs",
    "annotation_path": "./data/ucf101/annotation/ucf101_01.json",
    "result_path": "./data/ucf101/results",
    "dataset": "ucf101",
    "n_classes": 400,
    "n_finetune_classes": 101,
    "sample_size": 112,
    "sample_duration": 16,
    "initial_scale": 1.0,
    "n_scales": 5,
    "scale_step": 0.84089641525,
    "train_crop": "corner",
    "learning_rate": 0.1,
    "momentum": 0.9,
    "dampening": 0.9,
    "weight_decay": 0.001,
    "mean_dataset": "activitynet",
    "no_mean_norm": false,
    "std_norm": false,
    "nesterov": false,
    "optimizer": "sgd",
    "lr_patience": 10,
    "batch_size": 100,
    "n_epochs": 200,
    "begin_epoch": 81,
    "n_val_samples": 3,
    "resume_path": "./data/ucf101/results/save_80.pth",
    "pretrain_path": "./data/ucf101/pretrained/resnet-18-kinetics.pth",
    "ft_begin_index": 4,
    "no_train": false,
    "no_val": false,
    "test": false,
    "test_subset": "val",
    "scale_in_test": 1.0,
    "crop_position_in_test": "c",
    "no_softmax_in_test": false,
    "no_cuda": false,
    "n_threads": 30,
    "checkpoint": 10,
    "no_hflip": false,
    "norm_value": 1,
    "model": "resnet",
    "model_depth": 18,
    "resnet_shortcut": "A",
    "wide_resnet_k": 2,
    "resnext_cardinality": 32,
    "manual_seed": 1,
    "scales": [
        1.0,
        0.84089641525,
        0.7071067811803005,
        0.5946035574934808,
        0.4999999999911653
    ],
    "arch": "resnet-18",
    "mean": [
        114.7748,
        107.7354,
        99.475
    ],
    "std": [
        38.7568578,
        37.88248729,
        40.02898126
    ]
```

**Comment by kenshohara** [comment]
I cannot find what wrong points are...
How about the accuracy of train.log and val.log?

**Comment by slighting666** [comment]
@bandontseng 
Hi, I got the accuracy of val.log, but I want to calculate the accuracy of video level, where can I get the val.json file and how to use eval_ucf101.py?
